wbooks/src/app/components/FilterInput/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from 'react';
+import { View, TextInput, Image, TouchableOpacity } from 'react-native';
+import { useDispatch } from 'react-redux';
+import icSearchPlaceholder from '@assets/general/ic_search_placeholder.png';
+import close from '@assets/general/close.png';
+import BookActions from '@redux/book/actions';
+import useDebounce from '@hooks/useDebounce';
+
+import styles from './styles';
+
+const delay: number = 300;
```

Comment:
2 things:

* Again constants must be un UPPERCASE
* Unnecessary TS interface

Reply:
Done!